microsoft-365/admin/create-groups/office-365-groups-expiration-policy.md

```diff
@@ -73,7 +76,20 @@ Group owners will only be notified via email. If the group was created via Plann
 
 30 days before the group expires, the group owners (or the email addresses that you specified for groups that don't have an owner) will receive an email allowing them to easily renew the group. If they don't renew it, they'll receive another renewal email 15 days before expiration. If they still haven't renewed it, they will receive one more email notification the day before expiration.
 
-If for some reason none of the owners or admins renew the group before it expires, the admin can still restore the group for up to 30 days after expiration. For details see: [Restore a deleted Microsoft 365 group](https://docs.microsoft.com/microsoft-365/admin/create-groups/restore-deleted-group).
+If for some reason none of the owners or admins renew the group before it expires, and automatic renewal does not occur due to the group not meeting the requirements to be automatically renewed, the admin can still restore the group for up to 30 days after expiration. For details see: [Restore a deleted Microsoft 365 group](https://docs.microsoft.com/microsoft-365/admin/create-groups/restore-deleted-group).
+
+## How automatic renewal fits into the group expiration timeline
+
+Automatic renewal checks occur starting 40 days in advance from the group's expiration date. The service will continually check for activity from this point until the group expires, is manually renewed, or meets the requirements for automatic renewal and is thus renewed. If the service determines the group has been active at any point during the group's lifecycle, the service will begin the automatic renewal process. For example, if activity occurred 60 days from expiration, the activity check performed 40 days from expiration would indicate the group is active, and the automatic group renewal process would immediately begin.
+
+This also means that if no activity is detected within the group's lifecycle when, 40 days from expiration, the automatic renewal service first begins searching for activity meeting the requirements for automatic renewal, the group may not be automatically immediately renewed 40 days from expiration. For example, if the group is inactive until 35 days from renewal, the automatic renewal process would not occur 40 days from expiration, and would instead begin 35 days from renewal.
```

Comment:
Can we rephrase as follows:
On the other hand if there is no activity in the group 30 days prior to expiry then group owners will receive an e-mail requesting manual renewal of the group. Similar e-mail notifications will also be sent 15 days and 1 day prior to expiry. Simultaneously, auto-renewal checks will continue to occur and will auto-renew the group anytime an activity is detected up to 48 hours prior to group expiry.